packages/koenig-lexical/demo/DemoApp.jsx

```diff
@@ -187,6 +187,20 @@ function DemoComposer({editorType, isMultiplayer, setWordCount}) {
                 containerRef.current.scrollTop = containerRef.current.scrollHeight;
             }
         }
+
+        // when clicking between cards, put focus on the next card
+        const clickedOnDragAndDrop = (event.target.closest('[data-koenig-dnd-draggable]') !== null) || event.target.hasAttribute('data-kg-draggable'); // could also look at first child having data-kg-card
```

Comment:
Minor thing but would it be better to use `[data-lexical-decorator="true"]`? That would cover situations where the drag and drop reorder plugin isn't used.